flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/common/config/PulsarClientFactory.java

```diff
@@ -153,7 +153,7 @@ public static PulsarClient createClient(PulsarConfiguration configuration) {
 
     /**
      * PulsarAdmin shares almost the same configuration with PulsarClient, but we separate this
-     * create method for directly creating it.
+     * creating method for directly use it.
```

Comment:
Nit: I am having a hard time understanding this doc string. Can you maybe rephrase it?

Reply:
OK.